src/components/ArticlesCard.jsx

```diff
@@ -0,0 +1,21 @@
+import { useNavigate } from "react-router-dom";
+import { articleCardHover, articleCardHoverEnd } from "../scripts/articleCardHover";
+
+const ArticlesCard = ({ article }) => {
+    const navigate = useNavigate();
+
+    const goToArticleHandler = (id) => {
+        navigate(`/articles/${id}`);
+    }
+
+    return <li className="articles-item" id={`articles-item${article.article_id}`} tabIndex="0" onClick={ () => goToArticleHandler(article.article_id)} onKeyDown={(event) => {
+        if (event.key === 'Enter') goToArticleHandler(article.article_id);
+    }} onMouseEnter={ () => articleCardHover(article.article_id)} onMouseLeave={() => articleCardHoverEnd(article.article_id)}>
+        <img src={article.article_img_url} alt={`${article.title} image`}></img>
+        <h3>{article.title}</h3>
+        <p>{article.topic}</p>
+        <p className="date">Posted: {Date(article.created_at)}</p>
```

Comment:
I would consider whether there is a better way of displaying the created_at in a more readable way for users. Once set up, you could extract it as a util function, test it individually, and then reuse it across your app